src/scenes/Partners/Detail/PartnerDetail.tsx

```diff
@@ -127,15 +129,26 @@ const PartnerHeader = ({ partner, editPartner }: PartnerViewEditProps) => {
   );
 };
 
+const StatusIcon = ({ isActive }: { isActive: Maybe<boolean> | Nil }) =>
```

Comment:
Why `Nil`? Asking because I haven't seen it before.

Reply:
Nil covers null or undefined and I noticed that undefined was a possibility per the lint check, so I thought to use Nil to cover both.

Reply:
The `Maybe` helper already adds `null` and `undefined`, just FYI

Reply:
when i change it back to null, i get the lint error on the isActive property, so not sure how you are able to do it with just null

Reply:
I'd prefer `boolean | Nil` to be consistent across our repos. `Maybe` is not ours and we don't really reference it anywhere.

Reply:
For historical reference, I was importing the wrong `maybe` helper.

Reply:
just seeing the above conversation; do you want me to remove the Maybe completely?

Reply:
hopefully this last push works

Reply:
@CarsonF up to you if you're ok with that or if we change it. `Maybe` comes form the graphql schema. It was used in another place too

Reply:
LGTM